src/crypto/ocb_internal.cc

```diff
@@ -522,8 +552,8 @@ struct _ae_ctx {
 	uint64_t KtopStr[3];                   /* Register correct, each item  */
     uint32_t ad_blocks_processed;
     uint32_t blocks_processed;
-    AES_KEY decrypt_key;
-    AES_KEY encrypt_key;
+    ocb_aes::KEY decrypt_key;
+    ocb_aes::KEY encrypt_key;
     #if (OCB_TAG_LEN == 0)
```

Comment:
In a separate commit, we probably should clean up OCB_TAG_LEN == 0, since we will never define that in a useful mosh.